scripts/cli-migrations/docker-entrypoint.sh

```diff
@@ -72,7 +72,8 @@ if [ -d "$HASURA_GRAPHQL_MIGRATIONS_DIR" ]; then
     cp -a "$HASURA_GRAPHQL_MIGRATIONS_DIR/." "$TEMP_MIGRATIONS_DIR/migrations/"
     cd "$TEMP_MIGRATIONS_DIR"
     echo "endpoint: http://localhost:$HASURA_GRAPHQL_MIGRATIONS_SERVER_PORT" > config.yaml
-    echo "show_update_notification: false" >> config.yaml
```

Comment:
@arvi3411301 Why do you think this didn't work on the CLI?

Reply:
@shahidhk we are checking for `show_update_notification` in global config not in project config

Reply:
@arvi3411301 Okay!